components/create-collective/CreateCollectiveForm.js

```diff
@@ -237,11 +262,19 @@ class CreateCollectiveForm extends React.Component {
                         <Field
                           as={StyledInputGroup}
                           {...inputProps}
-                          prepend="opencollective.com"
+                          prepend="opencollective.com/"
                           placeholder={placeholders.slug}
                         />
                       )}
                     </StyledInputField>
+                    {values.name.length > 0 && (
+                      <SubText>
+                        Suggested URL :{' '}
+                        <Span style={{}} onClick={changeSlug}>
+                          {`/${suggestedSlug()}`}
+                        </Span>
+                      </SubText>
+                    )}
```

Comment:
Rather than displaying that, I would prefer that we prefill the `URL` field (if it hasn't been touched). @znarf what do you think?

Reply:
Do you mean prefill it with the suggested slug?

Reply:
@Betree do you mean the `URL` field ?

Reply:
@davejon1337 I think @Betree meant the URL field yes.

@davejon1337 While I like your approach and it looks good, we can't move forward with that without involving #design.

Reply:
I understand.